packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -304,7 +304,8 @@ test.describe.serial('Errors', () => {
 
 	test('Root error falls back to error.html (unexpected error)', async ({ page }) => {
 		await page.goto('/errors/error-html');
-		await page.click('button:text-is("Unexpected")');
+		await Promise.all([page.waitForNavigation(), page.click('button:text-is("Unexpected")')]);
```

Comment:
couldn't you write these with `clicknav`?
```suggestion
		await clicknav('button:text-is("Unexpected")');
```